### PR TITLE
ADD : stackedit.io under online editors

### LINKED ISSUE #906: [Add] stackedit.io under online editors
### Site URL

https://stackedit.io/

### Description of the above provided site

**_stackedit.io_** is the best online `md` and `mdx` editor available out in market/web.
**Even Visual Studio Code (VS Code) does not have built-in support for MDX (Markdown with JSX) files.**

So it helps in online coding and preview of md and mdx on the go.
It's essential as it supports mdx which is not supported by any markdown editor in market

### Category

Frontend

### Sub Category

Online Code Editors

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by dev-AshishRanjan**
## Fixes Issue #906 

Closes #906 

## Changes proposed
Added data for StackEdit under Frontend -> online-code-editors 

> StackEdit is a user-friendly online markdown editor with real-time collaboration and cloud storage integration, making writing > and editing documents effortless and accessible from anywhere.

## Screenshots
Screenshot 2023-06-05 011245

## Note to reviewers
The code added is : 
```
{
    "name": "StackEdit",
    "description": "StackEdit is a user-friendly online markdown editor with real-time collaboration and cloud storage integration, making writing and editing documents effortless and accessible from anywhere.",
    "url": "https://stackedit.io/",
    "category": "frontend",
    "subcategory": "online-code-editors"
  },
```

### CODE REVIEW

**Comment by dev-AshishRanjan** [comment]
Hi @rupali-codes and @CBID2 ,
I have done my part, but unable to appoint reviewers

Can you check this PR

File: database/frontend/online-code-editors.json [hunk 1]
```diff
@@ -27,6 +27,13 @@
     "category": "frontend",
     "subcategory": "online-code-editors"
   },
+  {
+    "name": "StackEdit",
+    "description": "StackEdit is a user-friendly online markdown editor with real-time collaboration and cloud storage integration, making writing and editing documents effortless and accessible from anywhere.",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:32]: [Typo]
```suggestion
    "description": "StackEdit is a user-friendly online Markdown editor with real-time collaboration and cloud storage integration, making writing and editing documents effortless and accessible from anywhere.",
```

File: database/frontend/online-code-editors.json [hunk 2]
```diff
@@ -27,6 +27,13 @@
...
+    "url": "https://stackedit.io/",
+    "category": "frontend",
+    "subcategory": "online-code-editors"
+  },
   {
     "name": "Sololearn",
     "description": "It's multi-language online code editor test and share your code snippets.",
```
Comment (CBID2) [inline database/frontend/online-code-editors.json:39]: [Proofreading]
``suggestion
    "description": "It's multi-language online code editor where you can test and share your code snippets.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @dev-AshishRanjan! :) I pointed out some typos and proofreading errors that could be improved.

File: database/frontend/online-code-editors.json (hunk 1 above)
Comment (dev-AshishRanjan) [reply database/frontend/online-code-editors.json]: Sorry for the typo
I fixed it

File: database/frontend/online-code-editors.json (hunk 2 above)
Comment (dev-AshishRanjan) [reply database/frontend/online-code-editors.json]: It done

**Review by rupali-codes — APPROVED** [review]
lgtm

**Comment by dev-AshishRanjan** [comment]
Hi @CBID2 ,
I have made the requested changes,
Can you check/review the PR.

**Comment by rupali-codes** [comment]
merging the PR since all the changes suggested by @CBID2 were made.

**Pull request merged**